docs/security-legal-pii/security/terms.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Legal and Compliance Documents"
 sidebar_order: 1
-description: "Learn about Sentry's Terms of Service, Privacy Policy, Data Processing Addendum, California Consumer Privacy Act Addendum, and Business Associate Amendment. Sentry also provides a way to self-serve most legal and compliance documentation for your organization."
+description: "Learn about Sentry's Terms of Service, Privacy Policy, Data Processing Addendum, and Business Associate Amendment. Sentry also provides a way to self-serve most legal and compliance documentation for your organization."
```

Comment:
```suggestion
description: "Learn about Sentry's Terms of Service, Privacy Policy, Data Processing Addendum, and Business Associate Amendment."
```
Removing this since this is the next sentence in the docs :)